.storybook/main.ts

```diff
@@ -1,20 +1,20 @@
-import type {StorybookConfig} from '@storybook/core-common';
+import type {StorybookConfig} from '@storybook/types';
 
-type Main = {
-    managerHead: (head: string) => string;
-} & StorybookConfig;
-
-const main: Main = {
-    stories: ['../src/**/*.stories.mdx', '../src/**/*.stories.@(js|jsx|ts|tsx)'],
-    addons: ['@storybook/addon-essentials', '@storybook/addon-a11y', '@storybook/addon-react-native-web'],
```

Comment:
Would you like to share your consideration to remove this addon `@storybook/addon-react-native-web`? Is it because there's no entry command in `package.json` to run storybook on native platforms (no usage on native platforms) ?

Reply:
That's correct. I wasn't able to find this being used anywhere.